frontend_v2/src/app/components/home/home.component.ts

```diff
@@ -94,9 +94,11 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
 
   /**
    * Set SUBSCRIBE_FORM to this.components after view initialization
+   * (<any>window).twttr.widgets.load() this load Twitter Feed
```

Comment:
@haldendmello can you remove this comment? Rest of the PR LGTM

Reply:
ok

Reply:
@Ram81 I have removed the comment.